pom.xml

```diff
@@ -166,7 +166,6 @@
     <commons.collections.version>3.2.2</commons.collections.version>
     <scala.version>2.12.10</scala.version>
     <scala.binary.version>2.12</scala.binary.version>
-    <scalafmt.parameters>--diff --test</scalafmt.parameters>
```

Comment:
I think this is the problem. `parameters` is no longer available in the plugin. Without it, I think it formats all files and fails. I don't see a way to have the plugin not do this, hm.

Reply:
Oh, that sounds like a serious regression of `scalafmt`.

Reply:
Well, the plugin more than scalafmt. We could pursue a workaround, but maybe not worth it; I'm not sure how much this is used. 

Really, the current state of this PR isn't great, as it no longer does anything useful. I'll need to either delete it or revert and move it behind a Scala 2.12 profile. Any opinion?

Reply:
Well, the plugin more than scalafmt. We could pursue a workaround, but maybe not worth it; I'm not sure how much this is used. 

Really, the current state of this PR isn't great, as it no longer does anything useful. I'll need to either delete it or revert and move it behind a Scala 2.12 profile. Any opinion?

Reply:
I'm in favor of trying to keep this functionality, since "nit: 4 spaces" is not a welcoming first experience for new contributors.

It looks like the plugin stopped using the cli interface to scalafmt, which is why parameters aren't accepted anymore.  I've opened https://github.com/SimonJPegg/mvn_scalafmt/issues/48  to see if the plugin author has any suggestions.